src/MapperInterface.php

```diff
@@ -15,9 +15,11 @@ interface MapperInterface
      * Maps an object to an instance of class $to, provided a mapping is
      * configured.
      *
+     * @template T of object
      * @param array|object $source
      *   The source object.
      * @param string $targetClass
+     * @psalm-param class-string<T> $targetClass
```

Comment:
`@psalm-` tag should be presented after param description, otherwise `@psalm-` will catch description from "main" tags

ie:
```php
     * @param string $targetClass
     *   The target classname.
     * @psalm-param class-string<T> $targetClass
```

Reply:
I don't understand how that's a problem... Doesn't Psalm ignore descriptions since they're after the parameter name?

Reply:
IDE reads the description only from `@param` for a quick doc

Example with phpstorm:
![image](https://user-images.githubusercontent.com/12499813/110934931-b7aefc80-82fc-11eb-9531-2a10a2df6d78.png)
And with swapped lines:
![image](https://user-images.githubusercontent.com/12499813/110934989-d1504400-82fc-11eb-95a5-238f18a63df9.png)

Reply:
Oh ok! So the issue is for other tools, understood. I'm on it.